### PR TITLE
Rename custom identifiers

### PR DESCRIPTION
**Pull request by johanholmerin**
Prevent collision of custom identifiers(e.g. keyframes) by renaming them.

### CODE REVIEW

File: compiler/package.json [hunk 1]
```diff
@@ -42,7 +42,8 @@
     "postcss": "6.0.17",
     "postcss-discard-duplicates": "2.1.0",
     "postcss-js": "1.0.1",
-    "postcss-nest-atrules": "0.1.3"
+    "postcss-nest-atrules": "0.1.3",
+    "postcss-reduce-idents": "^4.0.1"
```
Comment (giuseppeg) [inline compiler/package.json]: it seems that this plugin is deprecated now

**Review by giuseppeg — COMMENTED** [review]
hi I am terribly sorry about the late review. I think I have notifications disabled for this repo :(

I think that this feature is not strictly necessary as you can transform the files with PostCSS or another preprocessor before running DSS eg. https://dss-lang.com/webpack/#with-postcss

**Comment by johanholmerin** [comment]
Without this styles are not fully scoped. This is how CSS modules works.

File: compiler/package.json (hunk 1 above)
Comment (johanholmerin) [reply compiler/package.json]: No, it has just been moved to the cssnano monorepo.

**Comment by giuseppeg** [comment]
@johanholmerin there are some (unrelated?) linter errors, do unit tests pass? If you don't have time or want to fix the linter errors I'll try to do that when I have some time and then merge this PR.

The repo and packages' dependencies are out to date because the project is stale but I can release a new version with this fix too. FWIW I am working on a new version of this https://github.com/giuseppeg/style-sheet which is more like a baseline CSS in JS solution but I will then write an adapter to write styles in a CSS Modules/DSS way i.e. in files

**Comment by johanholmerin** [comment]
Yes, tests pass.

**Pull request closed without merging**